content/2-templating/5-dom-ref/emberPolaris/input-focused.gjs

```diff
@@ -0,0 +1,10 @@
+import Component from "@glimmer/component";
+import { modifier } from "ember-modifier";
+
+export default class InputFocused extends Component {
+  autofocus = modifier((element) => element.focus());
```

Comment:
this should be defined outside the class, and then we don't need a class at all!

Reply:
TIL!